variables.tf

```diff
@@ -68,4 +68,34 @@ variable "db_iops" {
   description = "The requested number of I/O operations per second that the DB instance can support."
   type        = number
   default     = 1000
+}
+
+variable "bamboo_license" {
+  description = "Bamboo license."
+  type        = string
+  sensitive   = true
+}
+
+variable "bamboo_admin_username" {
+  description = "Bamboo system administrator username."
+  type        = string
+  sensitive   = true
```

Comment:
`bamboo_admin_username`, `bamboo_admin_display_name`, and `bamboo_admin_email_address` are not sensitive